lib/makwa/interaction.rb

```diff
@@ -74,5 +83,18 @@ def indent
       lvl = [0, calling_interactions.count].max
       "  " * lvl
     end
+
+    private
+
+    def run
+      return self.result = nil unless valid?
+
+      self.result = run_callbacks(:execute) do
+        execute
+      rescue ::Makwa::Interaction::Interrupt => e
+        errors.backtrace = e.errors.backtrace || e.backtrace
+        errors.merge!(e.errors)
+      end
+    end
```

Comment:
If we go with the alternative implementation of `Makwa::Interaction::Interrupt` as shown above, then we should be able to remove this re-implementation of ActiveInteraction::Base#run.